bluemira/geometry/tools.py

```diff
@@ -487,6 +487,10 @@ def _offset_wire_discretised(
     """
     from bluemira.geometry._deprecated_offset import offset_clipper
 
+    if not open_wire:
```

Comment:
Not sure how the `open_wire` parameter combines with the note "Can only handle closed wires". A little bit confusing for me. Probably renaming it as "force_closure" (or similar) would be more explanatory? Similar comment for the signature of `offset_wire`.
Moreover, the method should raise an error if the wire is not closed and, in this case, `open_wire` is `True`.

Reply:
Agree to clean this up... I probably need to update my environment as I ran this 10 times without any failure on my machine at work. It fails first time in the CI with an error I've never seen :/